src/app/models/prediction.py

```diff
@@ -38,14 +47,22 @@ async def get_prediction(self, user_name) -> list[dict]:
 
         """
         async with self.session:
-            query: Select = select(dbPrediction)
-            query = query.select_from(dbPrediction)
-            query = query.where(dbPrediction.name == user_name)
+            prediction_db: dbPrediction = aliased(dbPrediction, name="prediction_db")
+            query: Select = select(prediction_db)
+            query = query.select_from(prediction_db)
+            query = query.where(
+                text("prediction_db.name = :name").bindparams(name=user_name)
+            )  # prevent sql injection

```

Comment:
please revert this change

Reply:
need to use bind params on any user inputted fields to avoid sql injection

Reply:
using the orm avoids sql injection, bind param are only relevant if you are writing raw sql like you do with text("")

Reply:
resolve with this commit - 
[prediction user_name to name, remove bind, revert list match, fix test cases](https://github.com/Bot-detector/public-api/pull/25/commits/d6ea27becdde54111f6d60eab2770e3c49e73839)